### PR TITLE
Feature/pm ios25

### PR DESCRIPTION
**Pull request by brandonshin**
Adding ability to set styles

### CODE REVIEW

File: Sources/Kanna/libxmlHTMLNode.swift [hunk 1]
```diff
@@ -119,6 +122,36 @@ internal final class libxmlHTMLNode: XMLElement {
...
+    fileprivate var internalStyles: [String: String]?
+    var styles: [String : String]? {
+        get {
+            // Cached lookup for performance
+            if let i = internalStyles {
+                return i
+            }
+            
+            if let style = self["style"] {
+                let styleItems = style.split(separator: ";")
```
Comment (bigshahan) [inline Sources/Kanna/libxmlHTMLNode.swift:138]: There are weird edge cases that break this, specifically with regards to malformed styling. THe correct way to implement this is using a lexer parser.

This should be noted, and unit test this, including in malformed css test cases, such as:

`color: red;; background: blue;`
`color:: red;`
`font-family: "FontName;"; color: yellow`

**Review by bigshahan — APPROVED** [review]
Left a comment, but not a fan of the approach used. Add in requested test cases before merging.
